pkgs/development/libraries/libvmaf/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, intltool, libtool, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "libvmaf";
+  version = "1.3.15";
+
+  src = fetchFromGitHub {
+    owner = "netflix";
+    repo = "vmaf";
+    rev = "v${version}";
+    sha256="10kgcdf06hzhbl5r7zsllq88bxbyn282hfqx5i3hkp66fpq896d2";
+  };
+
+  nativeBuildInputs = [ autoconf automake intltool libtool pkgconfig ];
+  outputs = [ "out" "dev" ];
+  doCheck = true;
+
+  makeFlags = [ "INSTALL_PREFIX=${placeholder "out"}" ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/Netflix/vmaf";
+    description = "Perceptual video quality assessment based on multi-method fusion (VMAF).";
```

description [should not include a period at the end](https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes):
```suggestion
    description = "Perceptual video quality assessment based on multi-method fusion (VMAF)";
```